packages/shared/src/components/post/write/CreatePostButton.tsx

```diff
@@ -43,20 +44,32 @@ export function CreatePostButton({
     return !allowedToPost;
   };
 
+  const href =
+    link.post.create + (squad && allowedToPost ? `?sid=${squad.handle}` : '');
+
+  const Wrapper = compact ? LinkWithTooltip : React.Fragment;
```

Comment:
We have ConditionalWrapprr for this actually.
It has some side effects on rerenders but i think with your evaluation its ok

Reply:
thx for the pointer 👍